src/schema/v2/schema.ts

```diff
@@ -114,6 +115,7 @@ const OptionalFieldDirective = new GraphQLDirective({
 })
 
 const rootFields = {
+  algolia: Algolia,
```

Comment:
Rather than add this at the root, I might put it under `system` (as that already has some 'internal' token fetchers and such).